timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResult.vue

```diff
@@ -36,8 +36,17 @@ limitations under the License.
       </v-col>
 
       <v-col cols="1">
-        <div class="ml-3">
-          <small>{{ Object.keys(analyzer.data.timelines).length }} </small>
+        <div v-if="isActive" class="ml-1">
+          <v-progress-circular
+            :size="20"
+            :width="1"
+            indeterminate
+          >
+            <small>{{ Object.keys(analyzer.data.timelines).length }}</small>
+          </v-progress-circular>
```

Comment:
The progress-circular seems not to be triggered here during my tests:

If I get this code right, this should address point 4 in #2799 , but in my tests I was not able to get the circle showing any activity.
![image](https://github.com/google/timesketch/assets/99879757/fde1b202-b1df-40ec-ba6e-5c523b8c076b)

A second problem I noticed and that is visible in the screenshot above: The progress circle is not always displayed for the timeline level either. Most times it works, but here I had still 2 feature extraction analyses running for the last 2 timelines.